app/Policies/SlotPolicy.php

```
@@ -0,0 +1,50 @@
+namespace App\Policies;
+
+use App\Models\Enums\UnitPermission;
+use App\Models\Slot;
+use App\Models\User;
+
+class SlotPolicy
+{
+    /**
+     * Determine whether the user can view any models.
+     */
+    public function viewAny(?User $user): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)